channels/alpha

```diff
@@ -20,8 +20,10 @@ spec:
     requiredVersion: 1.4.2
   kopsVersions:
   - range: ">=1.5.0-alpha1"
-    recommendedVersion: 1.5.0-beta1
-    #requiredVersion: 1.5.0-beta1
+    recommendedVersion: 1.5.0-beta2
```

Comment:
I _think_ this makes sense.  The channel is how you soon you generally want to get stuff.  So if you want to be an early adopter of k8s 1.6, we might put that into the alpha channel.  We normally wouldn't propose beta releases of kops, except that our current recommended version of kops for people that want to install k8s 1.5 is beta2.

So in general I think most people will stick with the stable channel, which probably not get 1.6 until the inevitable point releases.  (e.g. I'm actually not unhappy that we're only recommending 1.5 now).  The alpha channels would have it basically on release, and if there were some new features that kops needed for 1.6 then that might mean we would recommend (or even require) a kops alpha or beta version.